server/controllers/userAuthController.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable indent */
+import moment from 'moment';
+import uuid from 'uuid';
+import 'babel-polyfill';
+import jwt from 'jsonwebtoken';
+import db from '../db/tables';
+import dba from '../db/index';
+import userAuthHelper from '../validation/userAuth';
+
+/**
+ * 
+ */
+class userAuthControllerClass {   
```

Comment:
Please remove the `Class`, it's obvious that it's a class. Just use `UserAuthController`. Also, classes should start with Uppercase (their file names too).
```suggestion
class UserAuthController {   
```